client/src/components/PlanShowPage.js

```diff
@@ -14,18 +17,83 @@ const PlanShowPage = (props) => {
     }
   }
 
+  const fetchUser = async () => {
+    const currentUser = await getCurrentUser()
+    setUser(currentUser)
+  }
+
   useEffect(() => {
-    fetchPlan()
+    fetchPlan(),
+      fetchUser()
```

Comment:
even if we were to keep `fetchUser`, I wouldnt have the comma here. Just keep them on two lines